gen_cert.py

```diff
@@ -85,14 +85,18 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
-
+    utc_zone = get_timezone('UTC')
+    to_zone = get_timezone(timezone)
+    
     if calling_date_parameter:
-        date_value = format_datetime(calling_date_parameter, 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = strptime(calling_date_parameter, '%y-%m-%d')
     elif configured_date_parameter == "ROLLING":
-        date_value = format_datetime(datetime.datetime.today(), 'MMMM d, y', tzinfo=timezone, locale=locale)
+        date = datetime.datetime.today()
```

Comment:
since the date can be initialized in three different ways we don;t want to initialize any of the dates on generation...